base_station/gui/src/components/DriveVelDataH.vue

```diff
@@ -75,9 +75,9 @@
       </thead>
       <tbody>
           <tr>
-              <td class = "tableElement">{{drive_state_data.leftState}}</td>
-              <td class = "tableElement">{{drive_state_data.rightState}}</td>
-              <td class = "tableElement">{{drive_state_data.rightState}}</td>
+              <td class = "tableElement">{{drive_state_data.frontState}}</td>

```

Comment:
does this render correctly? /if there are gui changes include screenshots next time

Reply:
Yes it renders correctly and sure